crates/mipsy_web/src/pages/main/app.rs

```diff
@@ -102,51 +104,51 @@ pub fn render_app() -> Html {
         let filename2 = filename.clone();
         use_effect_with_deps(
```

Comment:
All these changes are just me removing an unsafe and probably shouldnt be in this PR

Reply:
:hollow: excellence in design!!!!